src/js/item.ts

```diff
@@ -64,6 +70,31 @@ export function extractItems(
   return [];
 }
 
+
+// TODO not the best place for this method, should rearrange and possibly change the callsite where this is used
+export function getCategoriesForProduct(scheduler: IRequestScheduler, productUrl: string): Promise<string> {
```

Comment:
do we need to export it - given we've put category in the interface, why do we need to share the logic used to populate it?